docs/proposals/controller/README.md

```diff
@@ -0,0 +1,38 @@
+# Cluster Controller
+
+The Cluster Controller is an intermediary between the client and scheduler, its main task is to compare two states: the one requested by the client and the existing one reported by the scheduler and saved in the database. Based on the comparison results, the controller sends requests to the scheduler API in order to transform the current state into the requested one.
+
+### Services provided:
+
+HTTP API (OpenAPI 3.0 compliant) to connect with CLI
+gRPC client and API for the scheduler
+DATABASE access and control
+
+### Scope
+
+The Cluster Controller is responsible for:
+Receiving requests from the CLI (Controller External API)
+Receive updates on the instances status from scheduler (Controller Internal API)
+Store current cluster state in the database
+Constantly compare current state with the state requested by client
```

Comment:
What does constantly mean here? The client will send a request, and the controller will synchronously service that request. Describing this as a constant comparison between 2 states makes it look more complex than it probably is :)

Reply:
Okay, we changed the phrase. We only need to compare states when a new client request is received or when the scheduler reports an unexpected change, not constantly.